axon-server/security/access-control.md

```diff
@@ -2,86 +2,146 @@
 
 As Axon Server is an event store and may contain sensitive data it is always a good practice to enable access control in production and production-like environments. Enabling access control will require applications to provide a token when accessing Axon Server services \(both through gRPC and HTTP\), and require users to login to the dashboard.‌
 
-To enable access control in Axon Server add the following property to `axonserver.properties`:
+To enable access control in Axon Server \(SE/EE\) add the following property to `axonserver.properties`:
 
 ```text
 axoniq.axonserver.accesscontrol.enabled=true
 ```
 
 ## Axon Server Standard Edition <a id="axon-server-standard-edition"></a>
 
-Axon Server Standard Edition has a basic access control mechanism, where applications that provide a valid token can perform all operations. For users there are two groups, users that can login to the dashboard but cannot manage users and users with ADMIN role, who are allowed to manage users.‌
+Axon Server Standard Edition provides a basic access control mechanism for 
+
+* Client Applications connecting to Axon Server SE.
+* Users accessing the Axon Server UI Console.
+
+### Axon Framework Client Applications
+
+Applications need to provide a valid token can perform all operations. For users there are two groups, users that can login to the dashboard but cannot manage users and users with ADMIN role, who are allowed to manage users.‌
 
 The token that applications must provide to use Axon Server is defined in the `axonserver.properties` file.
 
 ```text
 axoniq.axonserver.accesscontrol.token=[token]‌
 ```
 
-The token that you set here must be used by all applications connecting to Axon Server.‌
+The token that you set here must be used by all Axon Framework Applications connecting to Axon Server. The access token can be setup in the client using the property `axon.axonserver.token=[Token]`
 
-You can create users either through the command line or through the dashboard web application. The initial user has to be created through the command line as you would not be able to login tp the dashboard. To do this execute the following command:
+If you are using the REST APIs, you can specify the token in the HTTP requests via the following header:
 
 ```text
-$ java -jar axonserver-cli.jar register-user -S        http://[node]:[port] -u [username] -r [roles] -t [token]
+AxonIQ-Access-Token: my-token-value-here
 ```
 
-The token that you must use here is the token you have set in the `axonserver.properties` file. In this case you will be prompted for a password \(you can pass the password in the command line using the -p option\). Valid roles are READ and ADMIN.‌
+### **Users - Axon Server SE UI Console**
+
+You can create users either through the command line or through the dashboard web application. The _**initial user**_ has to be created through the command line as you would not be able to login tp the dashboard. 
```

Comment:
```suggestion
You can create users either through the command line or through the dashboard web application. The _**initial user**_ has to be created through the command line as you would not be able to login to the dashboard otherwise.
```